back/config/tenant_templates/base.yml

```diff
@@ -1,4 +1,23 @@
 models:
+  home_page:
+    -
+      top_info_section_enabled: true
+      top_info_section_multiloc: {}
+      bottom_info_section_enabled: true
+      bottom_info_section_multiloc: {}
+      events_enabled: false
+      projects_enabled: true
+      projects_header_multiloc: {}
+      banner_avatars_enabled: true
+      banner_enabled: true
+      banner_layout: "full_width_banner_layout"
+      banner_signed_in_header_multiloc: {}
+      cta_signed_in_text_multiloc: {}
+      cta_signed_in_type: "no_button"
+      banner_signed_out_header_multiloc: {}
+      banner_signed_out_subheader_multiloc: {}
+      cta_signed_out_text_multiloc: {}
+      cta_signed_out_type: "sign_up_button"
```

Comment:
We actually don't need to specify all values, as is done here.

The problem is that we iterate over these values with `each`, meaning that we could include any single value and this will still create exactly the same record (using default values, and NULL values), but will fail if no value(s) provided, with
```
Loading of the template failed.
undefined method `each' for nil:NilClass
```
So, I decided if we need to specify at least one value, we may as well be explicit and specify all.